data/gegno/gegno outfits.txt

```diff
@@ -0,0 +1,807 @@
+		"Gegno Civilian"
+	plural "R03 Warforge Batteries"
+	cost 61700
+	mass 41
+	thumbnail "outfit/gbatterymed"
+	"energy capacity" 4469
+	"outfit space" -41
+	description `Warforge Batteries have proved their usefulness over the years as large, sturdy battery packs. They support the equally large volleys that are fired from Gegno ships, and help maintain an excess of power for engine usage.`
+
+
+outfit "R04 Crusade Battery"
+	category "Power"
+	licenses
+		"Gegno Civilian"
+		"Vi Evocati"
+		"Vi Centurion"
+		"Vi Lord"
+	plural "R04 Crusade Batteries"
+	cost 136000
+	mass 83
+	thumbnail "outfit/gbatterylarge"
+	"energy capacity" 8885
+	"outfit space" -83
+	description `As the largest battery mounted on energy-hungry Gegno ships, the Crusade Battery holds a large amount of power, more than it would take a small generator a decent amount of time to produce. However, the Gegno are inefficient at increasing density with size, causing these batteries to fall behind similarly sized options.`
+
+
+outfit "C3 Scrap Cell"
+	category "Power"
+	licenses
+		"Gegno Civilian"
+	cost 54000
+	mass 25
+	thumbnail "outfit/vicoretiny"
+	"energy generation" 1.4
+	"heat generation" 2.2
+	"outfit space" -25
+	description `The Scrap Cell is a small radiothermal generator that produces a limited amount of energy. It is only used in tandem with the Gegno's smaller-sized vessels and internals.`
+
+
+outfit "C7 Brawl Cell"
+	category "Power"
+	licenses
+		"Gegno Civilian"
+	cost 68100
+	mass 39
+	thumbnail "outfit/vicoresmall"
+	"energy generation" 2.5
+	"heat generation" 3.9
+	"outfit space" -39
+	description `Unlike other C-class cores, C7 Brawl Cells are mounted horizontally. This design still serves the Gegno well, though it will likely be phased out in favor of larger generators as they become more compact and efficient.`
+
+
+outfit "C17 Warzone Core"
+	category "Power"
+	licenses
+		"Gegno Civilian"
+	cost 2472000
+	mass 87
+	thumbnail "outfit/vicoremed"
+	"energy generation" 14.1
+	"heat generation" 38.3
+	"outfit space" -87
+	description `As did humans, the Gegno began developing nuclear technology once their cities grew massive enough to require it. The Warzone Core is a medium-sized nuclear fission reactor that is a suitable power source for most of the Gegno's large ships, though it looks even more discomforting than a human one.`
+
+
+outfit "C27 Campaign Core"
+	category "Power"
+	licenses
+		"Gegno Civilian"
+		"Vi Evocati"
+		"Vi Centurion"
+		"Vi Lord"
+	cost 8145000
+	mass 145
+	thumbnail "outfit/vicorelarge"
+	"energy generation" 22.3
+	"heat generation" 62.9
+	"outfit space" -145
+	description `In order to power the largest of vessels along with their power-guzzling systems and engines, the Gegno designed the Campaign Fusion Reactor from earlier prototypes during their colony ship era. It's still used to this day and is efficient enough to satisfy the requirements of the largest of Gegno warships, but just looking at it raises safety concerns.`
```

Comment:
> Campaign Fusion Reactor

Since the name of the actual outfit is the Campaign Core, idk if it makes sense to capitalize "fusion reactor" here.